### PR TITLE
Add OpenJDK configure reference to build instructions

### PR DESCRIPTION
**Pull request by pshipton**

### CODE REVIEW

File: doc/build-instructions/Build_Instructions_V11.md [hunk 1]
```diff
@@ -150,6 +150,10 @@ bash configure --with-boot-jdk=/home/jenkins/bootjdks/jdk11
 ```
 :warning: The path in the example --with-boot-jdk= option is appropriate for the Docker installation. If not using the Docker environment, set the path appropriate for your setup, such as "<my_home_dir>/bootjdk11" as setup in the previous instructions.
 
+:pencil: Configuring and building is not specific to OpenJ9 but uses the OpenJDK build infrastructure with OpenJ9 added in.
+Many other configuration options are available, including options to increase the verbosity of the build output to include command lines (`LOG=cmdlines`) or other info or debug information.
+For more information see the OpenJDK build instructions https://htmlpreview.github.io/?https://raw.githubusercontent.com/openjdk/jdk/master/doc/building.html#troubleshooting .
```
Comment (keithc-ca) [inline doc/build-instructions/Build_Instructions_V11.md:155]: This should link to the jdk11 documentation, not jdk.
Suggest that "OpenJDK build instructions" name the link:
```
For more information see the [OpenJDK build instructions](https://htmlpreview.github.io/?https://raw.githubusercontent.com/openjdk/jdk/master/doc/building.html#troubleshooting).
```

Comment (pshipton) [reply doc/build-instructions/Build_Instructions_V11.md]: Updated.

**Comment by pshipton** [comment]
Tweaked the working a little.

**Comment by pshipton** [comment]
and again.

**Comment by pshipton** [comment]
@0xdaryl fyi

File: doc/build-instructions/Build_Instructions_V11.md [hunk 2]
```diff
@@ -150,6 +150,10 @@ bash configure --with-boot-jdk=/home/jenkins/bootjdks/jdk11
 ```
 :warning: The path in the example --with-boot-jdk= option is appropriate for the Docker installation. If not using the Docker environment, set the path appropriate for your setup, such as "<my_home_dir>/bootjdk11" as setup in the previous instructions.
 
+:pencil: Configuring and building is not specific to OpenJ9 but uses the OpenJDK build infrastructure with OpenJ9 added.
+Many other configuration options are available, including options to increase the verbosity of the build output to include command lines (`LOG=cmdlines`), more info or debug information.
+For more information see [OpenJDK build troubleshotting](https://htmlpreview.github.io/?https://raw.githubusercontent.com/openjdk/jdk11u/master/doc/building.html#troubleshooting) .
```
Comment (keithc-ca) [inline doc/build-instructions/Build_Instructions_V11.md:155]: nit: Please remove the space before the period (here and and in other, similar additions).

Comment (pshipton) [reply doc/build-instructions/Build_Instructions_V11.md]: Updated.

**Review by keithc-ca — APPROVED** [review]

**Comment by 0xdaryl** [comment]
Thanks for adding this.

FYI, "troubleshooting" is misspelled throughout (as "troubleshotting").

**Comment by pshipton** [comment]
Created https://github.com/eclipse-openj9/openj9/pull/16067

**Pull request merged**